samples/dashboards/demo/election/demo.js

```diff
@@ -56,21 +56,32 @@ async function setupDashboard() {
         },
         components: [
             {
```

Comment:
It would be good to standardize the way of writing brackets, so I propose:
```ts
components: [{
    ...
}, {
    ...
}]
```
instead of
```ts
components: [
    {
         ...
    },
    {
         ...
    }
]
```